include/yaml-cpp/version.h

```diff
@@ -0,0 +1,17 @@
+#ifndef YAML_CPP_VERSION_H
+#define YAML_CPP_VERSION_H
+
+#if defined(_MSC_VER) ||                                            \
+    (defined(__GNUC__) && (__GNUC__ == 3 && __GNUC_MINOR__ >= 4) || \
+     (__GNUC__ >= 4))  // GCC supports "pragma once" correctly since 3.4
+#pragma once
+#endif
+
+#define YAML_CPP_VERSION_MAJOR 0
+#define YAML_CPP_VERSION_MINOR 5
+#define YAML_CPP_VERSION_PATCH 2
```

Comment:
if that's the case then should the equivalent variable in the top-level CMakeLists.txt also be 3? especially as this is the variable that is used to generate the version_test that this file is checked against.